apps/services/user-profile/src/app/v2/me-user-profile.controller.ts

```diff
@@ -74,7 +74,11 @@ export class MeUserProfileController {
           fields: Object.keys(userProfile),
         },
       },
-      this.userProfileService.patch(user.nationalId, userProfile),
+      this.userProfileService.patch(
```

Comment:
I would update the signature of the `patch` function to accept the `user` object instead of passing `nationalId` and `audkenniSimNumber` as two separate arguments.